src/index.ts

```diff
@@ -661,6 +690,48 @@ function makeSourceMap(
   };
 }
 
+/**
+ * This method maps the newly generated @param{sourceMap} to the input source map.
+ * This is required when ts-loader is not the first loader in the Webpack loader chain.
+ * If the input source map is not set, then the newly generated source map is returned.
```

Comment:
Is this line still true? It looks like that's handled elsewhere now?

Reply:
Thanks, that was an old comment.